wiki/en/Running-a-Server.md

```diff
@@ -201,15 +201,15 @@ Jamulus can be run in Server mode from the desktop. This gives you a graphical u
 
 **None**: By default, you will not be connected to a Directory and will be running in unregistered mode. [Read these instructions](#running-an-unregistered-server) to have other people connect to your Server in this mode.
 
-**Genre**: To allow other people to see your Server on one of the built-in Public Directories, select your desired genre Directory. You should see a confirmation message saying whether your Server has registered successfully. If not, and you leave your Server running, it will keep trying to register until a free slot becomes available.  
+**Genre**: To allow other people to see your Server on one of the built-in Directories, select your desired genre Directory. You should see a confirmation message saying whether your Server has registered successfully. If not, and you leave your Server running, it will keep trying to register until a free slot becomes available.  
```

Comment:
Inconsistent with an earlier change from "built in" to "Public".  Stick with "Public".

Reply:
As per my comment above, I think use of the term "Public" may be confusing.

Reply:
I think the term "Public" is correct.  If its "confusing" then its usage hasn't been made clear enough in the manual.

Reply:
You're right, it's not been made clear, which is why I'm debating it :-)

"Public" may well be correct, but if we just use "built-in Directories" to refer to Directories that appear in the software, then that avoids questions about "non-built in", since those terms don't usually pair up in people's minds like "public and private".  

As it stands, if you read part of the manual and encounter a reference to "Public Directories", you'd have to read the rest to understand that "Private" doesn't exist. It seems an unnecessary distraction (particularly for those used to the previous usage in the context of Servers).

Reply:
OK, so rather than
> To allow other people to see your Server on one of the built-in Directories
say
> To allow other people to see your Server on one of the Directories built into the Client

Reply:
It might not be directly releate, but I think we should move away from Genre too and use something like "List" instead. Otherwise, we'd have too many equal-ish terms:

Genre, Genre Server, Directory, List, Directory Server,... all kind of mean something similar.